experimental/adaptive-tool/README.md

```diff
@@ -78,38 +114,35 @@ This is an example of a typical `launch.json`:
 }
 ```
 
-`debugServer` refers to the port bot runs on.
+`debugServer` refers to the port bot runs on. The default value is `4712`.
 
 ### Start bot runtime and complete initialization
 
-Start a bot runtime. For example, start a bot project: `todobot` in [SampleBots](https://github.com/microsoft/botbuilder-dotnet/tree/main/tests/Microsoft.Bot.Builder.TestBot.Json). Then make sure the debugger port has been registered in `BotFrameworkHttpAdapter` with `UseDebugger` method.
-
-There are several ways to initialize a bot, and [BotFremawork Emulator](https://github.com/microsoft/BotFramework-Emulator) is a typical approach. Open the emulator and attach it to the bot to finish the initialization.
+1. Start a bot runtime. For this example we'll start the bot project `TodoBot` in [SampleBots](https://github.com/microsoft/botbuilder-dotnet/tree/main/tests/Microsoft.Bot.Builder.TestBot.Json).
+1. Make sure the debugger port has been registered in `BotFrameworkHttpAdapter` with the `UseDebugger` method.
+1. There are several ways to initialize a bot, and [BotFramework Emulator](https://github.com/microsoft/BotFramework-Emulator) is a typical approach. Open the Emulator and attach it to the bot to finish the initialization.
 
 ### Debug the runtime
```

Comment:
```suggestion
### Debug the bot
```
We're really not debugging the bot runtime/executable so much as we're debugging the resource files (.lu/.lg/.dialog).